README.md

```diff
@@ -101,25 +101,25 @@ cd core-primitives
 npm install
 ```
 
-### Adding or updating primitives
+## Adding or updating primitives
 
-### 1. Create a new branch
+## 1. Create a new branch
```

Comment:
These numbered items should probably be `###` headings, because I think they're supposed to be under "Adding or updating primitives."

Reply:
Nice catch, thanks 👍